DiceKit/Die.swift

```diff
@@ -134,3 +134,27 @@ extension Die: ExpressionType {
 public func == (lhs: Die, rhs: Die) -> Bool {
     return lhs.sides == rhs.sides
 }
+
+// MARK: - CustomDebugStringConvertible
+
+extension Die: CustomDebugStringConvertible {
+    
+    public var debugDescription: String {
+        get {
+            return "Die(\(sides))"
+        }
+    }
+    
+}
+
+// MARK: - CustomStringConvertible
+
+extension Die: CustomStringConvertible {
+    
+    public var description: String {
+        get {
+            return "d\(sides)"
```

Comment:
If negative should this instead be `d(-12)`? I think it would look better for that case.